janusgraph-core/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 JanusGraph Authors
```

Comment:
Could you delete this file?

Reply:
Sorry about this one! I deleted this file.